css/style.css

```diff
@@ -1486,36 +1485,40 @@ a.fa-linkedin {
 .slider:before {
   position: absolute;
   content: "";
-  height: 15px;
-  width: 15px;
-  left: 3px;
-  bottom: 1.5px;
-  background-color: white;
+  height: 18px;
+  width: 18px;
+  bottom: -.8px;
+  background: white url("../assets/Images/mode/sunny.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  box-shadow: 0 0px 15px #2020203d;
   -webkit-transition: .4s;
   transition: .4s;
+  right: 0.89rem;
 }
 
 input:checked + .slider {
-  background-color: var(--bg2-color);
-}
-
-@media only screen and (max-width: 991px)
-{
-  input:checked + .slider {
-  background-color: rgb(106, 68, 241);
-  }
+  background-color: #ccc;
 }
    
 input:focus + .slider {
-  box-shadow: 0 0 1px #111;
+  box-shadow: 0 0 1px #2196f3;
 }
 
 input:checked + .slider:before {
-  transform: translateX(18px);
+  -webkit-transform: translateX(24px);
+  -ms-transform: translateX(24px);
+  transform: translateX(24px);
+  background: white url("../assets/Images/mode/night.png");
```

Comment:
As I have mentioned before, please name the  sub-directories  inside the assets directory judiciously so that it becomes easy for us to document this and debug later. Please make it ```dark_mode_switch/night.png```